README.md

```diff
@@ -170,29 +195,35 @@ This object represents the centroid of the Item Geometry.
 There are several projection extension fields with potentially overlapping functionality. This section attempts to 
 give an overview of which ones you should consider using. They fit into three general categories:
 
-- **Description of the coordinate reference system:** [EPSG code](#projepsg) is recommended, but it is just a reference to known
-projection information. [WKT2](#projwkt2) and [PROJJSON](#projprojjson) are two options to fully describe the projection information. 
-This is typically done for projections that aren't available or fully described in the [EPSG Registry](https://epsg.org/). 
+- **Description of the coordinate reference system:** [proj:code](#projcode) is recommended, but it is just a 
+reference to known projection information. [WKT2](#projwkt2) and [PROJJSON](#projprojjson) are two options to 
+fully describe the projection information. 
+
+This is typically done for projections that aren't available or fully described in a known registry 
+(e.g., [EPSG Registry](https://epsg.org/) or [IAU Registry](http://www.opengis.net/def/crs/IAU/2015)). 
+
 For example, the MODIS Sinusoidal projection does not have an EPSG code, but can be described using WKT2 or PROJJSON.
 - **Description of the native geometry information:** STAC requires the geometry and bounding box, but they are only available
-in lat/long (EPSG:4326). But most remote sensing data does not come in that projection, so it is often useful for clients to have 
+in lat/long (EPSG:4326, IAU_2015:30100, IAU_2015:49900, etc.). But most remote sensing data does not come in 
+that projection, so it is often useful for clients to have 
 the geometry information ([geometry](#projgeometry), [bbox](#projbbox), [centroid](#projcentroid)) in the coordinate reference system
 of the asset's data, so it doesn't have to reproject (which can be lossy and takes time). 
 - **Information to enable cataloging of data without opening assets:** Often it is useful to be able to construct a 'virtual layer',
 like GDAL's [VRT](https://gdal.org/drivers/raster/vrt.html) without having to open the actual imagery file. [shape](#projshape) and
 [transform](#projtransform) together with the core description of the CRS provide enough information about the size and shape of
 the data in the file so that tools don't have to open it.
 
-For example, the GDAL implementation [requires](https://twitter.com/EvenRouault/status/1419752806735568902) the following fields: 
-1. `proj:epsg`, `proj:wkt2` or `proj:projjson` (one of them filled with non-null values)
+For example, the GDAL implementation [requires](https://twitter.com/EvenRouault/status/1419752806735568902) 
+the following fields: 
+1. `proj:wkt2` or `proj:projjson` (one of them filled with non-null values)
```

Comment:
Could/should we update the GDAL implementation to use `proj:code` after this lands?

Reply:
I guess for some time we'll need to support both, proj:epsg and proj:code.